kong/plugins/zipkin/schema.lua

```diff
@@ -12,6 +12,7 @@ return {
                              between = { 0, 1 } } },
                                         { default_service_name = { type = "string", default = nil } },
           { include_credential = { type = "boolean", required = true, default = true } },
+          { traceid_bytes = { type = "integer", required = true, default = 16, one_of = { 8, 16 } } },
```

Comment:
i might consider calling this something like `traceid_byte_count` or such, for clarity; calling it `*_bytes` to me implies the value should contain actual bytes, not that it's a configurable measure of length.

pedantic $0.02 :)

Reply:
Done!